src/app/_constants/actions.ts

```diff
@@ -19,6 +19,12 @@ export const ACTIONS_BULK_EDIT = {
   QUARANTINE_DOCUMENTS: 'CREATE_QUARANTINE_ORDER_DOCUMENTS',
 };
 
+export const ACTIONS_VIEW_OPTION = {
```

Comment:
the name is too generic, maybe ACTIONS_EVENTS_VIEW_OPTIONS would be more suited (we have view options in multiple places, not only events page)

Reply:
Changed.